packages/scoutgame/src/points/dividePointsBetweenBuilderAndScouts.ts

```diff
@@ -43,50 +43,29 @@ export async function dividePointsBetweenBuilderAndScouts({
     throw new InvalidInputError('Invalid rank provided. Must be a number greater than 0');
   }
 
-  const nftPurchaseEvents = await prisma.nFTPurchaseEvent.findMany({
-    where: {
-      builderEvent: {
-        week: {
-          lte: week
+  // Calculate the total number of NFTs purchased by each scout
```

Comment:
by removing the NFT history query here, we can be much faster if we just retrieve nft purchases for all builders ta once